3rd/src/041Summary.py

```diff
@@ -0,0 +1,40 @@
+#! python3
+
+#ランダムで値を取り出す為の準備
+import random #
+
+#エラーが発生した際に別処理へ遷移させたい箇所をtry～exceptで囲む。
+try:
+
+#おみくじ用ファイルを読み込む
+    daikiti = open("daikiti.txt", "r",encoding="utf-8")
+    kiti    = open("kiti.txt", "r",encoding="utf-8")
+    kyo     = open("kyo.txt", "r",encoding="utf-8")
+
+#読み込んだおみくじ用ファイルを辞書型でまとめる
+    input = {"大吉":daikiti,

```

Comment:
```suggestion
    omikuji_dic = {"大吉":daikiti,
```
`input`は組み込み関数なのでできるだけ使用しないほうがいいと思います。